### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by cynthiajoan**
[replace this line]: # (Describe your changes in detail.)
***

[replace this line]: # (Describe your testing in detail.)
***

### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [ ] New feature. A non-breaking change which adds functionality.
- [ ] Other, such as a build process or documentation change.
***

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,140 @@
...
-* **PROTOBUF_SRC_ROOT_FOLDER**: Protobuf root directory
+``` bash
+python scripts/build_scripts/build_zips.py –platform=<target platform, pick one from linux,macos,windows,ios,android>
+```
 
-> Note:<br/>
-> &nbsp;&nbsp;&nbsp;**UNITY_ROOT_DIR** is a recommended setting for building
-> unity, else CMake will make an effort to auto find unity.
+Expected output artifact is
+[Repo Root]/<platform_unity, eg macos_unity>/firebase_unity-<version>-<platform>.zip
```
Comment (a-maurice) [inline README.md:71]: version and platform don't seem to render correctly, might need to use something other than < and >

File: README.md [hunk 2]
```diff
@@ -1,101 +1,140 @@
...
-cmake .. -DFIREBASE_INCLUDE_MONO=ON -DFIREBASE_CPP_SDK_DIR=../../firebase-cpp-sdk
-make -j 8
+python scripts/build_scripts/build_zips.py —-platform=<target platform> --targets=<lib1> -–targets=<lib2>
 ```
 
-> Note:<br/>
-> &nbsp;&nbsp;&nbsp;There are build helper scripts in the root folder that will
-> build the most common variants and can be used as examples on how to build
-> firebase unity.
+> **Note:** library name: analytics, auth, crashlytics, database, dynamic_links, firestore, functions, installations, messaging, remote_config, storage
```
Comment (a-maurice) [inline README.md:85]: Supported library names:

**Review by a-maurice — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cynthiajoan) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (cynthiajoan) [reply README.md]: done

**Pull request merged**